docs/docs-content/clusters/edge/upgrade-behavior.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_label: "Edge Cluster Upgrade Behavior"
+title: "Edge Cluster Upgrade Behavior"
+description: "Learn about how Palette Edge responds to upgrades of the cluster profile."
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["edge", "architecture"]
+---
+
+When you update an active Edge cluster's profile, Palette will upgrade the active cluster to the latest version of the
+profile. Depending on the nature of the change, implementing an upgrade might involve repaving a cluster, rebooting a
+cluster, restarting services, or doing nothing. For more information about cluster repaves, refer to
+[Repave Behavior and Configurations](../cluster-management/node-pool.md#repave-behavior-and-configuration).
+
+## Upgrade Behaviors
+
+A cluster could respond to an upgrade in several ways.
+
+| Upgrade Behavior     | Description                                                                                                                                             |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Repave               | Repaves all nodes in the cluster, starting with the control plane.                                                                                      |
+| Reboot               | Reboots all nodes in the cluster, starting with the control plane                                                                                       |
+| Service restart      | Restarts specific services in all nodes in the cluster, starting with the control plane                                                                 |
+| Configuration reload | Updates the configurations without needing to restart services or the node. This is only triggered when you update `stage.reconcile.*` in the OS layer. |
+| No-op                | Acknowledges the update request but does nothing.                                                                                                       |
```

Comment:
@lennessyy the term `no-op` most tech savy people understand it means no operations but maybe we should explain that in description?